frontend/src/EmbeddedApp.tsx

```diff
@@ -68,11 +75,15 @@ function EmbeddedApp({ basePath, ...p }: EmbeddedProps) {
 
     setup(p);
 
+    if (!p.isConsoleReadyToMount) {
```

Comment:
We have to make sure we support this in every environment before we create a install pack with this console version

Reply:
Agreed